packages/rlc-common/src/metadata/buildApiExtractorConfig.ts

```diff
@@ -5,16 +5,16 @@ import { Project } from "ts-morph";
 import { RLCModel } from "../interfaces.js";
 
 export function buildApiExtractorConfig(model: RLCModel) {
-  const generateMetadata = Boolean(model.options?.generateMetadata);
-  if (!generateMetadata) {
-    return;
-  }
-  const { generateTest, packageDetails } = model.options || {};
+  let { generateTest, packageDetails, isModularLibrary } = model.options || {};
+  // Take the undefined as true by default
+  generateTest = generateTest === true || generateTest === undefined;
```

Comment:
Can we have some automatical check if the tests exist or not ?

Reply:
We do this check in emitter side, https://github.com/Azure/autorest.typescript/pull/1967/files#diff-3d0e467e1b9edbc8377b86fbeaac71ce5bc0ddc8260bee5e56a777360cf19398R236-R242.

In common layer we also need to check if we need to generate test relevant stuff so we have the dependency of `generateTest`.